assets/src/components/template-inserter/edit.css

```diff
@@ -6,8 +6,7 @@
 	margin-right: 10px;
 }
 
-#amp-story-controls .editor-inserter__amp-inserter span:first-of-type,
-.amp-story-controls-reorder span:first-of-type {
+#amp-story-controls .editor-inserter__amp-inserter span:first-of-type {
```

Comment:
^ Should we remove this, too, to show the tooltip for the Template Inserter? It doesn't show for me when leaving this in at least.


Unfortunately, I don't remember anymore why was hiding the first `span` necessary but I think it might have been related to styling this here (which seems to look a bit broken sometimes when scrolling, not related to this PR though):
![Screen Shot 2019-05-17 at 2 04 40 AM](https://user-images.githubusercontent.com/3294597/57894795-8a9c7a80-7848-11e9-9fb0-867af4339601.png)

Reply:
Guess who just tested in dev tools and didn't remove both lines 🤦‍♂